go-controller/pkg/util/util.go

```diff
@@ -27,6 +27,24 @@ const (
 	// access to local service
 	LocalNetworkName = "locnet"
 
+	// OVNClusterRouter is the name of the distributed router
```

Comment:
is util the right place to have all these constants? Seems a little odd to me, but I'm good with centralizing them to the same place that we currently use.

Reply:
Not sure, it seems PR: https://github.com/ovn-org/ovn-kubernetes/pull/1754/commits/3b7906fa734192b01a53b7f3591b335348b416ec adds that to `pkg/config` 

I don't have a particularly strong opinion on the matter, but for this PR I would need it removed from `pkg/ovn/gateway.go` as to be able to use it from: https://github.com/ovn-org/ovn-kubernetes/blob/7ba0a081d49dfeb42323e22a8381d9f0c0b6906e/go-controller/pkg/util/net.go#L42 (the change introduced by my second commit) without introducing a cyclic dependency in the code

Reply:
I think config seems like a better place as long as it doesn't introduce a dependency problem. I'll  leave it @JacobTanenbaum to follow up on that